15-components-lab_v16/src/00-synctext.jsx

```diff
@@ -1,14 +1,26 @@
-/**
- * Synchronised input fields
- *
- * Create a React component that shows 5 input fields
- * and synchronises their input:
- * Each time the value in one input changes,
- * all other input fields should show that updated value
- */
-
-import React from 'react';
-
-class MultiInput extends React.Component {
-}
-
+import React from 'react';
+
+
+class MultiInput extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = { textValue : "type sodsfmething..." };
+  }
+
+ write = (e) => {
+   this.setState({ textValue: e.target.value });

```

Comment:
Did you try `this.setState(() => ({...}))` (Passing a function) ? Did it work? Can you explain why?